conf/application.conf

```diff
@@ -149,3 +142,59 @@ microservice {
         }
     }
 }
+
+
+# Microservice specific config
+
+wiremock-port = 11111
+wiremock-port = ${?WIREMOCK_PORT}
+
+# TODO discover if this is needed
```

Comment:
If not needed, remove it and remove the TODO comment

Reply:
done